### PR TITLE
Feature/opinion modal adjustments

### PR DESCRIPTION
**Pull request by cfviotti**
Open the floodgates

### CODE REVIEW

File: src/static/js/components/opinionModal/view.js [hunk 1]
```diff
@@ -60,6 +67,15 @@ OpinionModalView.prototype.subscribers = function () {
   events.closeModal.subscribe(function() {
     events.closeOpinionModal.publish();
   });
+
+  events.showSubmissionCue.subscribe(function(opinion) {
+    self.showSubmissionCue(opinion);
+  });
+
+  events.showNextSuggestion.subscribe(function() {
+    self.nextOpinionElements.removeClass('-ripple -inactive');
```
Comment (msfernandes) [inline src/static/js/components/opinionModal/view.js]: Should these commands be a function?

**Review by msfernandes — CHANGES_REQUESTED** [review]

**Comment by msfernandes** [comment]
Should the opinionModal close when I click on body overlay?

File: src/static/js/components/opinionModal/view.js (hunk 1 above)
Comment (cfviotti) [reply src/static/js/components/opinionModal/view.js]: I believe I fixed that. Please review once more.

**Comment by cfviotti** [comment]
> Should the opinionModal close when I click on body overlay?

Are you asking if this should happen? Because it doesn't right now by design. For the opinion-modal you should only close it by clicking its closing button.

If you're actually saying that this is already happening, it shouldn't. Please show me the steps to replicate that if so.

**Comment by msfernandes** [comment]
taokey

**Pull request merged**